fickling/loader.py

```diff
@@ -1,25 +1,26 @@
 import pickle
 
+from fickling.analysis import Severity
 from fickling.fickle import Pickled
 
 
-# TODO Determine what the best defaults for block are
-def load(file, run_after_analysis=True, block=[3, 4, 5]):
+def load(
+    file,
+    max_acceptable_severity=Severity.LIKELY_UNSAFE,
+    print_results=False,
+    json_output_path=None,
+    *args,
+    **kwargs,
+):
     """Exposed as fickling.load()"""
     pickled_data = Pickled.load(file)
-    result = pickled_data.check_safety.severity
-    if result not in block:
-        # Apply a fallback in case of custom unpicklers unknown to the user
-        if run_after_analysis is True:
-            try:
-                return pickle.loads(pickled_data.dumps())
-            except Exception:  # noqa
-                raise ValueError(
-                    """The data could not be dumped and pickled.
-                                 Try `run_after_analysis=False` and run
-                                 the custom unpickling after"""
-                )
-        else:
-            return True
+    result = pickled_data.check_safety(
+        return_result=True, print_results=print_results, json_output_path=json_output_path
+    )
+    if result.severity < max_acceptable_severity:
+        # We don't do pickle.load(file) because it could allow for a race
+        # condition where the file we check is not the same that gets
+        # loaded after the analysis.
+        return pickle.loads(pickled_data.dumps(), *args, **kwargs)
     else:
         return False
```

Comment:
If the file is considered unsafe then we should raise an exception I think. That's what makes the most sense from the user perspective. Then they can write stuff like
```
import fickling
try:
    obj = fickling.load(untrusted_file.pkl)
except fickling.SafetyError as e:
    # Unsafe file
    print("File is unsafe!!!", e)
    abort()
# Safe file
obj.do_stuff()
```